app/actions/CommonActions.scala

```diff
@@ -52,9 +53,9 @@ object CommonActions {
   }
 
   implicit class MobileSupportRequest[A](val request: Request[A]) extends AnyVal {
-    def platform: Option[String] = request.getQueryString("platform") orElse request.session.get("platform")
-    def isAndroid: Boolean = platform.contains("android")
-    def isIos: Boolean = platform.contains("ios")
+    def platform: String = request.getQueryString("platform").orElse(request.session.get("platform")).getOrElse("web")
```

Comment:
`platform` is also derived from a request [here](https://github.com/guardian/contributions-frontend/pull/305/files#diff-0ad4a3f0bc037938eadbb8872d92a4b3R65). @alexduf could this method be used instead?